src/types/Props.ts

```diff
@@ -0,0 +1,6 @@
+import { User } from './User';
+import { Todo } from './Todo';
+
+export interface Props extends Todo {
+  user: User | null,
+}
```

Comment:
You can have a lot of Props:
```suggestion
export interface TodoWithUser extends Todo {
  user: User | null,
}
```